TCHAP_CHANGES.md

```diff
@@ -3,6 +3,9 @@ Changes in Tchap 1.99.3 (2021-xx-xx)
 
 Bugfix 🐛:
  - [Login Screen] An unexpected error on the password field #276
+
+Improvements 🙌:
+ - Remove irrelevant Matrix Id PR #294
```

Comment:
```suggestion
 - Remove irrelevant Matrix Id #294
```

Reply:
@giomfo ask me to put the PR to make a reference to the PR and not to the ISSUE.

Reply:
ACK